tests/kafkatest/utils/remote_account.py

```diff
@@ -29,4 +31,28 @@ def line_count(node, file):
     if len(out) != 1:
         raise Exception("Expected single line of output from wc -l")
 
-    return int(out[0].strip().split(" ")[0])
\ No newline at end of file
+    return int(out[0].strip().split(" ")[0])
+
+def scp(source_node, source_path, target_node, target_path):
+    """ Copy file from source node to destination node. Uses a temporary file
+    on local node, since there is no mechanism to copy directly between two remote nodes.
+    :param source_node: source node to copy from
+    :param source_path: path on the source node to copy from
+    :param target_node: destination node
+    :param target_path: path on the destination node to copy to
+    """
+    local_temp_dir = "/tmp"
```

Comment:
A little confusing to set `local_temp_dir = "/tmp"` since it in theory should never be set to `/tmp`

I think this line is actually not necessary since `local_temp_dir` can be set inside the try block, and be available to the surrounding scope